homework.md

```diff
@@ -86,28 +87,28 @@ You send the changes of your local repository to Github
 
 What is Github?
 
-A place for us to store and share our code.
+is a source control platform use when created code .is a store place where you can save your project
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+is to solve the error your change to the repository or stash your change later
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual studio Code
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+when things really tend to go south the revision history is a great resource to find out quickly where exactly things went wrong
 
```

Comment:
I believe we use commit messages to identify our changes. They provide information about what has changed and why.